facebook/delphiFacebook/R/contingency_write.R

```diff
@@ -109,15 +107,62 @@ add_geo_vars <- function(data, params, geo_type) {
         .data$county_fips
       )
   } else if (geo_type == "county") {
-    warning("county metadata not supported")
+    counties <- read_csv(
+      file.path(params$static_dir, "02_20_uszips.csv"),
+      col_types = cols(.default = "c")
+    ) %>% 
+      mutate(
+        fips = stri_pad(.data$fips, 5, pad="0")
```

Comment:
I usually use `sprintf("%05d", .data$fips)`, but I guess you've already imported `stri` so this is fine (and probably easier to understand)